app/(gcforms)/[locale]/(form administration)/form-builder/[id]/components/dialogs/RulesDialog/Rules.tsx

```diff
@@ -1,33 +1,32 @@
 "use client";
-import React, { useState } from "react";
-import PropTypes from "prop-types";
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "@i18n/client";
 
 import { Button } from "@clientComponents/globals";
 import { useTemplateStore } from "@lib/store/useTemplateStore";
-import { ModalProperties } from "@lib/store/useModalRulesStore";
-import { FormElementWithIndex } from "@lib/types/form-builder-types";
-import { ChoiceRule } from "@lib/formContext";
+import { ChoiceRule, getElementsWithRuleForChoice } from "@lib/formContext";
 import { ConditionalSelector } from "@formBuilder/components/shared/conditionals/ConditionalSelector";
 import { sortByGroups, sortByLayout } from "@lib/utils/form-builder";
 import { AddOther } from "@formBuilder/components/shared/conditionals/AddOther";
 
 import Markdown from "markdown-to-jsx";
+import { FormElementWithIndex } from "@lib/types/form-builder-types";
 
-export const ModalFormRules = ({
+export const Rules = ({
```

Comment:
Maybe we should call this RulesSelectors or something ... to better describe what the UI is

Reply:
Renamed to RulesForm 33b04179a37334558bd7784a0266c712c3edc635